app.js

```diff
@@ -31,13 +32,19 @@ var helper = function(path) {
   return require(appPath("app/helpers/" + path));
 };
 
+// set logger file
+if (app.get('env') === 'development') {
+  winston.level = 'debug';
+}
+winston.add(winston.transports.File, { filename: 'winston.log' });
```

Comment:
Set up Winston settings here. This will get the default singleton instances and these settings will apply to the `require('winston')` calls in the other files.